src/likes.js

```diff
@@ -0,0 +1,44 @@
+import { postLikes, getLikes } from './api';
+
+const itemsID = (element) => element.parentNode.parentNode.parentNode.id;
+
+const displayLikes = async () => {
+  const likesData = await getLikes();
+  const likesCount = document.querySelectorAll('.likes-count');
+
+  likesCount.forEach((likes) => {
+    const id = itemsID(likes);
+
+    likesData.forEach((item) => {
+      if (item.item_id === id) likes.innerHTML = item.likes;
+    });
+  });
+};
+
+const UpdateLike = async (target) => {
+  const likesTag = target.parentNode.lastElementChild;
+  let likes = Number(likesTag.textContent);
+  likes += 1;
+
+  likesTag.innerHTML = likes;
+};
```

Comment:
- [ ] It would be better to keep the function synchronous if not necessary. Currently, you don't have a blocking piece of code here which needs an async call. If yes please use the `await` keyword to handle the response. Let's fix that